pkg/reconciler/knativeserving/knativeserving_controller.go

```diff
@@ -201,6 +205,9 @@ func (r *ReconcileKnativeServing) install(instance *servingv1alpha1.KnativeServi
 		err = extensions.PreInstall(instance)
 		if err == nil {
 			err = r.config.ApplyAll()
+			if err != nil {
```

Comment:
Can we add a link / a description of why were doing this (to force our client to refresh its CRD discovery mappings)?

Reply:
Done.

Reply:
The root cause of this seems to be that Manifestival's RESTMapper doesn't refresh itself. The operator-sdk provides such a mapper.

I think https://github.com/jcrossley3/manifestival/blob/2d595bb74f312fb5f9228ade0de5dbf792840061/manifestival.go#L54 needs to change to said mapper and we might not need this workaround.

Reply:
Doh! Why didn't I think of that?!?! Good call. I'll take care of it!

Reply:
I have updated the `client-go` branch of manifestival to use a dynamic RESTMapper that will obviate the need for this workaround. There should be no need to refresh the client anymore.